src/main/java/de/zalando/aruha/nakadi/util/UUIDGenerator.java

```diff
@@ -0,0 +1,9 @@
+package de.zalando.aruha.nakadi.util;
+
+import java.util.UUID;
+
+public class UUIDGenerator {
+    public UUID randomUUID() {
+        return UUID.randomUUID();
+    }
```

Comment:
Having it as a separate class can be useful when creating tests (I see that Ricardo mocks it in the test)
So I now added usage of that class in another place as well.
The places where we generate UUID in tests I didn't touch because those are tests :)